ujt/constants.py

```diff
@@ -9,14 +9,9 @@
 
 from . import id_constants
 
-# configurable constants -- move this to a config file
-CLEAR_CACHE_ON_STARTUP = True
-REFRESH_TOPOLOGY_ON_STARTUP = True
-REFRESH_SLI_ON_INTERVAL = True
-CLIENT_SLI_REFRESH_INTERVAL_MILLIS = 30 * 1000  # ms
-SERVER_SLI_REFRESH_INTERVAL_SECONDS = 30
-
 # functional constants
+SLI_CACHE_TIMEOUT_SECONDS = 30
```

Comment:
This is provided as an argument to a decorator: `@cache.memoize(timeout=SLI_CACHE_TIMEOUT_SECONDS)` and decorators are run at load-time. So we can't enforce the parsing of this option before the decorator is run unless we place the parsing above the imports (bad). I'd rather leave this as a non-configurable option. 

(Asked John about this as well, he had no solution :( )

Reply:
can probably leave this explanation as a comment and it will be fine.